package sec01.exam01;

public class HumanException extends Exception {
	
	HumanException() {
		System.out.println("HumanException 생성자 실행");
	}

}
